persistence/sql/migrations/sql/20220825134336000000_delete_verification_token_without_flow_id.up.sql

```diff
@@ -0,0 +1,2 @@
+DELETE FROM identity_verification_tokens
+WHERE selfservice_verification_flow_id IS NULL;
```

Comment:
Why not add a migration that sets this column to `NOT NULL`?

Reply:
True, will add another migration.

Reply:
:+1: the rest looks very good

Reply:
Urgh, I added the migrations. Turned out to be a lot of work for what it is ^^😓